packages/node-integration-tests/suites/express/tracing/server.ts

```diff
@@ -8,7 +7,7 @@ const app = express();
 Sentry.init({
   dsn: 'https://public@dsn.ingest.sentry.io/1337',
   release: '1.0',
-  integrations: [new Sentry.Integrations.Http({ tracing: true }), new Tracing.Integrations.Express({ app })],
+  integrations: [new Sentry.Integrations.Http({ tracing: true }), new Sentry.Express({ app })],
```

Comment:
I wonder if this feels like a mismatch, where some are nested under `Integrations`, and others are now at the top level export.

@kamilogorek brings it up here as well: https://github.com/getsentry/sentry-javascript/pull/7511#discussion_r1141312322

cc @lforst

Reply:
Yep agree it looks a bit confusing. 

This PR keeps the previous functionality of lazily loading the optional integrations so they are not picked up by bundlers when using auto discovery. If we export them all under `Integrations` this lazy loading becomes more redundant. Using any optional integration will mean they all get bundled.

It suppose it depends on how much we care about treeshaking for node. Treeshaking is mildly important for the Electron SDK since it directly impacts app startup performance. However, as I don't know what size all these optional integrations bundle down to so it's hard to judge the impact.